src/test/java/io/zold/api/RtTransactionTest.java

```diff
@@ -58,9 +60,14 @@ public void amountIsNotYetImplemented() {
         new RtTransaction("amount()").amount();
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void prefixIsNotYetImplemented() {
-        new RtTransaction("prefix()").prefix();
+    @Test
+    public void returnsPrefix() {
```

Comment:
@paulodamaso let's add another test, where we check if an exception is thrown if the prefix field is missing.

Reply:
@paulodamaso let's also add another test where we check an exception is thrown if the prefix format has been violated (i.e. 8-32 base64 symbol characters).